locales/de/dashboard.ftl

```diff
@@ -43,19 +42,15 @@ modal-active-number-of-exposures-part-two = Wenn Sie beispielsweise 10 Offenlegu
 modal-active-number-of-exposures-part-three-all = Sobald sie behoben wurden, werden sie zur Gesamtzahl der behobenen Offenlegungen auf der Seite „Behoben“ hinzugefügt.
 modal-fixed-number-of-exposures-title = Über die Anzahl der behobenen Offenlegungen
 modal-fixed-number-of-exposures-all = Diese Tabelle enthält die Gesamtzahl der Datenlecks, die für alle E-Mail-Adressen behoben wurden, die Sie derzeit überwachen. Sobald Offenlegungen als behoben markiert werden, werden sie hier zur Gesamtsumme hinzugefügt.
-
 modal-cta-ok = OK
-modal-open-alt = Öffnen
-modal-close-alt = Schließen
-
+open-modal-alt = Modales . öffnen
```

Comment:
@flodolo Not sure if I have to do anything with this, and my German is mediocre at best, but this ` . ` seems wrong?

Reply:
Yeah, this doesn't look right. I'll fix it in Pontoon.

Reply:
Oh, never mind. Pretranslation is enabled for German in that project, so this is coming from Google Translate. It will be fixed by translators, likely in the next hours knowing Michael.
https://pontoon.mozilla.org/de/firefox-monitor-website/all-resources/?status=pretranslated&string=303076